handel/test/lifecycles/delete-test.ts

```diff
@@ -49,8 +49,9 @@ describe('delete lifecycle module', () => {
                 A: new UnPreDeployContext(serviceContext)
             });
             const serviceRegistry = new FakeServiceRegistry();
+            const opts: DeleteOptions = { debug: false, linkExtensions: false, yes: false, environments: ['dev'], accountConfig: '' };
```

Comment:
One more little thing, this module fails to compile for me locally because this line has "environments" still instead of "environment".